src/main/webapp/dev/DatePickerJsTest.js

```diff
@@ -0,0 +1,22 @@
+QUnit.module('datepicker.js');
+
+QUnit.test('getMinDateForPublishDate(visibleDate)', function(assert) {
+    assert.deepEqual(getMinDateForPublishDate(new Date('10/13/2014')), new Date('10/13/2014'), 'returns visibleDate');
```

Comment:
then startDate and publishDate are also parameters?
then should i change returns startDate(parameter)?